$OpenBSD: patch-src_tags_cpp,v 1.3 2015/06/19 16:23:33 dcoppa Exp $

commit d54de65990dbe852762e5292c5b27186d16fcc02
Author: Andrzej Rybczak <electricityispower@gmail.com>
Date:   Mon May 18 00:05:49 2015 +0200

tags: remove xiph tag DESCRIPTION as it's parsed as COMMENT by MPD

--- src/tags.cpp.orig	Sat May  2 21:16:11 2015
+++ src/tags.cpp	Fri Jun 19 16:29:53 2015
@@ -184,6 +184,8 @@ void writeXiphComments(const MPD::MutableSong &s, TagL
 	tag->removeField("TRACK");
 	// remove field DISC, some taggers use it as DISCNUMBER
 	tag->removeField("DISC");
+	// remove field DESCRIPTION, it's displayed as COMMENT
+	tag->removeField("DESCRIPTION");
 	writeXiph("TITLE", tagList(s, &MPD::Song::getTitle));
 	writeXiph("ARTIST", tagList(s, &MPD::Song::getArtist));
 	writeXiph("ALBUMARTIST", tagList(s, &MPD::Song::getAlbumArtist));
